packages/realm-server/server.ts

```diff
@@ -60,6 +67,23 @@ export class RealmServer {
             'Authorization, Content-Type, If-Match, X-Requested-With',
         })
       )
+      .use(async (ctx, next) => {
+        // Disable browser cache for all data requests to the realm server. The condition captures our supported mime types but not others,
```

Comment:
@habdelra Can you check if this comment makes sense? I didn't want to interfere with assets caching behaviour, so I limited this to our supported card MIME types.